### PR TITLE
css-box-model-Bishan-Webeshete

### PR DESCRIPTION
**Pull request by BishanWebeshete**
Screen Shot 2023-01-25 at 9 31 24 AM

### CODE REVIEW

File: css-box-model/styles.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+
+  /* your code here 👇 */
+  margin-left: 12px;
+  margin-right: 12px;
+  padding-left: 16px;
+  padding-right: 16px;
+  padding-top: 6px;
```
Comment (shawnkost) [inline css-box-model/styles.css:10]: You need to add a `padding-bottom` property as well

**Review by shawnkost — CHANGES_REQUESTED** [review]
Hey Bishan! One small change requested here

**Comment by BishanWebeshete** [comment]
Screen Shot 2023-01-25 at 9 45 49 AM

**Review by shawnkost — APPROVED** [review]
:+1:

**Pull request merged**